compiler/rustc_passes/src/reachable.rs

```diff
@@ -21,7 +24,9 @@ use rustc_privacy::DefIdVisitor;
 use rustc_session::config::CrateType;
 use rustc_target::spec::abi::Abi;
 
-fn item_might_be_inlined(tcx: TyCtxt<'_>, def_id: DefId) -> bool {
+/// Determines whether this (assumed to be reachable) item "needs cross-crate MIR", i.e. whether
+/// another crate needs to be able to access this items' MIR.
+fn needs_cross_crate_mir(tcx: TyCtxt<'_>, def_id: DefId) -> bool {
```

Comment:
I decided to introduce a new term: "needs cross-crate MIR". I think that makes this much easier to explain.

Reply:
Unfortunately that term doesn't work. :/ We don't actually cross-crate-need the MIR of `const fn`, we need to walk their body for other reasons...